docs/tutorial/file.rst

```diff
@@ -105,6 +105,14 @@ The channels have a few convenient methods, like `.plot()` which make it easy to
     plt.plot(f1x_timestamps, f1x_data)
 
 The `timestamps` attribute returns absolute values in nanoseconds.
+Note that `timestamps` cannot be converted to floating point without losing precision::
```

Comment:
Yeah, that's a good point. I've reworded it to:
```
The `timestamps` attribute returns the measurement time in nanoseconds since epoch (January 1st 1970, midnight UTC/GMT), not counting leap seconds.
Note that since these values are typically very large, they cannot be converted to floating point without losing precision::
```
Not sure whether the leap seconds bit makes it more confusing than its worth though.

Reply:
yeah, I would leave the leap seconds out, it's extraneous. the point is, it's a really freaking big number